src/dialogs/config-flow/step-flow-abort.ts

```diff
@@ -33,6 +53,36 @@ class StepFlowAbort extends LitElement {
     `;
   }
 
+  private async _handleMissingCreds() {
+    if (
+      !(await showConfirmationDialog(this, {
+        title: this.hass.localize(
```

Comment:
Can we add info here provided by core how you would get an application credential?

Reply:
Yes, i think it makes sense and that was the original motivation for going down this path. I originally thought it made the most sense to have this inline in the config flow like this:
https://github.com/home-assistant/core/pull/64680#issue-1111219737
and that is still the user experience I'd like to get back to.

From a UX perspective, my recommendation is that we add this on the same screen where the user is entering the application credentials, which is the page after the abort error, similar to what the screenshot looks like above.

However, that dialog does not *yet* support integration specific customizations. Originally I was thinking that the inline application credentials could also be entered in the config flow itself (with a duplicate set of boxes in the config flow that mirror what exists in the app creds panel), instead of redirecting in the frontend, given that config flows are already very easy to optimize. However, going with this frontend approach would imply that we need to extend that dialog with core driven customizations.